static/css/components/mybooks-menu.less

```
@@ -4,6 +4,25 @@
  *
  * http://localhost:8080/account/books/want-to-read
  */
+
+/* 'create a list' within sidebar */
+.sidebar-section {
+  .create {
+    text-align: center;
+  }
```

We can remove the `.create` element entirely and style the `widget-add` instead.

```suggestion
  .widget-add {
    text-align: center;
    padding: 6px 0;
  }
```